src/rev_ai/apiclient.py

```diff
@@ -322,8 +341,29 @@ def get_account(self):
 
         :raises: HTTPError
         """
-        url = urljoin(self.base_url, 'account')
-        response = self.session.get(url)
-        response.raise_for_status()
+        response = self._HTTPHandler(
+                      "GET",
+                      urljoin(self.base_url, 'account')
+        )
 
         return Account.from_json(response.json())
+
+    def _HTTPHandler(self, method, url, **kwargs):
+        """Wrapper method for initiating HTTP requests and handling potential
+            errors.
+
+        :param method: string of HTTP method request
+        :param url: string containing the URL to make the request to
+        :param (optional) **kwargs: potential extra arguments including header
+            and stream
+        :raises: HTTPError
+        """
+        response = self.session.request(method, url, **kwargs)
+
+        try:
+            response.raise_for_status()
+            return response
+        except HTTPError as err:
+            err.args = (err.args[0] +
```

Comment:
We should make sure response.content isn't null. Currently if they get a 500 or a 503 it may error